doc/lambdapi.bnf

```diff
@@ -24,10 +24,11 @@ QID ::= [UID "."]+ UID
               <inductive>)* ";"
             | "rule" <rule> ("with" <rule>)* ";"
             | "builtin" <stringlit> "≔" <qid_or_nat> ";"
+            | "coercion" <rule> ";"
```

Comment:
Why not coerce_rule to indicate that this is a rule, and follow the pattern we already have with unif_rule?

Reply:
No need to change coercion everywhere, just in the concrete syntax.